vanilla/vanilla
# Fix dropdown view profile link when Garden.Profiles.Edit is removed

**Issue**

When removing _Garden.Profiles.Edit_ from a role, the dropdown menu item from the mebox to **View** profiles disappears as well.

**Details**

When we build the dropdown menu we are checking if the user has _Garden.Profiles.Edit_ permissions for both the view and edit link.  We should check the respective permission for each link.

**To Test**

1) Role the edit profile permission on role.
2) Create a user with that role.
3) Login with the new user and try see if the view link is in the mebox dropdown menu

Closes vanilla/support#542
